main.go

```diff
@@ -297,6 +321,67 @@ func getSystemInfo() *SystemInfo {
 	return info
 }
 
+// Max returns the larger of x or y.
+func Max(x, y uint64) uint64 {
```

Comment:
Out of curiosity, we're not using the Min/Max native functions because they are `float64` specific? https://golang.org/pkg/math/#Min

Later edit: never mind me, I see below you're using the native ones and turn to these only for the `RSS` and `AvailableMemory` large values. LGTM!

Reply:
Yeah, this would be a good situation for generics, but it's not an option here.